src/main/java/nl/tudelft/contextproject/model/level/RoomTuple.java

```diff
@@ -0,0 +1,64 @@
+package nl.tudelft.contextproject.model.level;
+
+/**
+ * Room tuple, data container for passing around 2 rooms at the same time.
+ * 
+ * Should be used for starter and treasure rooms.
+ */
+public class RoomTuple {
+	
+	protected Room starterRoom;
+	
+	protected Room treasureRoom;
+	
+	/**
+	 * Constructor.
+	 * @param starterRoom
+	 * 			starter Room
+	 * @param treasureRoom
+	 * 			treasure Room
+	 */
+	public RoomTuple(Room starterRoom, Room treasureRoom) {
+		this.starterRoom = starterRoom;
+		this.treasureRoom = treasureRoom;
+	}
+
+	/**
+	 * Get starter room.
```

Comment:
please add a newline after this line, so the @ lines are seperated from the method description